app/components/views/HelpPage/TutorialsTab/index.js

```diff
@@ -2,6 +2,7 @@ import { FormattedMessage as T } from "react-intl";
 import { DescriptionHeader } from "layout";
 import { Subtitle } from "shared";
 import { Link } from "react-router-dom";
+import recentTicketsStyles from "../../HomePage/RecentTickets/RecentTickets.module.css";
```

Comment:
Please create a local css module and use it here, this breaks the whole idea behind css modules.
css module should be a component specific.